### ISSUE TITLE
Default "HelloWorld" app for "JavaScript Chome App (using Polymer paper elements)" broken

### ISSUE THREAD

**Issue by davedawkins** [GoogleChrome/chromedeveditor#3979]
Once the project is created, it says that it "will not run as is", and needs "Refactor for CSP" called on "bower_components".

However, this operation fails with:

> Error while refactorying bower_components for CSP
> NotFoundError: A requested file or directory could not be found at the time an operation was processed

I can get the "Refactor for CSP" completed if I remove "hydrolysis" and "marked_element" from bower_components.

Even, then I'm getting these errors in the console:
````
net::ERR_FILE_NOT_FOUND  ..../bower_components/core-icons/core-icons.html
net::ERR_FILE_NOT_FOUND  ..../bower_components/core-toolbar/core-toolbar.html
Refused to load the stylesheet '...family=Roboto...' because it violates ...
Refused to load the stylesheet '...family=Roboto+Mono...' because it violates ...
```
So it looks like core-toolbar and core-icons are missing. I've seen workarounds for the stylesheet, but I feel a demo app should work "out of the box".

**Comment by ussuri** [comment]
I'm working on a fix.

**Comment by davedawkins** [comment]
Thank you

**Comment by ron813c** [comment]
check your install location and try running the windows file in your power shell

**Comment by MingweiSamuel** [comment]
I also have this error. This should be a high-priority fix IMO, since it should work out of the box.

**Comment by eduardomazolini** [comment]
Is there any workaround? Some manual intervention? Until there is a definitive solution.

**Comment by jheiselman** [comment]
I would like to bump this for a resolution. I am just getting started with developing a Chrome App and so need a fully functional template to working from. I am on ChromeOS.

**Comment by mkayen** [comment]
+1 - Running into the same issue.

**Comment by jmetz** [comment]
+1

**Comment by ZanderBrown** [comment]
any workaround?

**Comment by iSuslov** [comment]
+1. Ridiculous, a year since this issue was reported

**Comment by jheiselman** [comment]
I'm convinced that they've given up on this effort. I got my Chrome App working. It doesn't look pretty and has zero design because that's what I wanted polymer for. I am not a designer. But, it will be the last Chrome App I write. If I need anything else on my Chromebook, I'll just put Linux on it because I am clearly not the target audience.

**Comment by gio1135** [comment]
I know how you feel.

**Comment by ZanderBrown** [comment]
I know Chrome Dev Editor has kinda been abandoned but the Chrome platform hasn't. Is it really that hard to make a working Hello World? I've given up on Chrome development. If this gets fixed I might come back but for now it's not worth the effort

**Comment by jheiselman** [comment]
Exactly. I am no good with design work. I am a single developer. I can't afford, nor do I have any inclination, to hire a designer. If this can't deliver on the promise of making design easier and more consistent, then I will abandon it until the Chrome team decides that this is something that is worthwhile investing time in, or another solution comes along.

**Comment by seanryanstephens** [comment]
Unfortunately, this project appears pretty much abandoned. Unless someone outside the original team is able to generate a fix for merge/pull, then it's a lost cause. ("Note: Chrome Dev Editor is no longer being actively developed. We'll do our best to merge your pull requests.") I'm a beginner programmer and hope to get proficient enough to work on CDE myself...

**Comment by gio1135** [comment]
I put this together so you don't have to compile Chrome Dev Editor in order to even use it. So far, I've only updated the JSLint. I'll put the installation instructions in the README.
https://github.com/gio1135/chromedeveditor
